src/brod_group_coordinator.erl

```diff
@@ -403,6 +410,10 @@ handle_call({commit_offsets, ExtraOffsets}, From, State) ->
 handle_call(Call, _From, State) ->
   {reply, {error, {unknown_call, Call}}, State}.
 
+handle_cast({update_topics, Topics}, State) ->
+  NewState0 = State#state{ topics = Topics},
+  {ok, NewState1} = stabilize(NewState0, 0, topics),
```

Comment:
Nitpick: according to our code convention, last variable assignment should not have trailing numbers. So this variable should be called `NewState` instead of `NewState1`.